Libs/Core/ctkUtils.cpp

```diff
@@ -109,17 +113,30 @@ const char *ctkValidWildCard =
 QStringList ctk::nameFilterToExtensions(const QString& nameFilter)
 {
   QRegExp regexp(QString::fromLatin1(ctkNameFilterRegExp));
+#if QT_VERSION >= 0x060000
+  int i = nameFilter.indexOf(regexp);
+#else
   int i = regexp.indexIn(nameFilter);
+#endif
   if (i < 0)
     {
     QRegExp isWildCard(QString::fromLatin1(ctkValidWildCard));
+#if QT_VERSION >= 0x060000
+    if (nameFilter.indexOf(isWildCard) >= 0)
+#else
     if (isWildCard.indexIn(nameFilter) >= 0)
+#endif
       {
       return QStringList(nameFilter);
       }
     return QStringList();
     }
+#if QT_VERSION >= 0x060000
+  QStringList captured = regexp.namedCaptureGroups();
+  QString f = captured[2];
```

Comment:
add a check that there are at least 3 strings in `captured` before trying to access the `captured[2]`